src/templates/static/get_started/binary_options_mt5.jsx

```diff
@@ -16,6 +16,12 @@ const BinaryOptionsForMT5 = () => (
                             { id: 'how-to-trade-binary',     href: '#how-to-trade-binary',     text: it.L('How to trade binary options on MT5') },
                         ]}
                     />
+
+                    <div className='center-text gr-padding-10 client_logged_out invisible'>
+                        <a className='button' href={it.url_for('new-account')}>
+                            <span>{it.L('Open a free account')}</span>
```

Comment:
as far as I see they wanted a button to open an MT5 account, not a binary account. the page needs to be updated, see the same button at the bottom of the get-stareted/metatrader. also why are we only adding to one of the sub pages of the get-started/metatrader? @ashkanx

Reply:
yes, the button should be for MT5 account

Reply:
@brandon-binary please add the mt5 account opening button to the bottom of the menu of all get started/metatrader sub pages then. better to use a common component in that case.